src/main/java/co/com/bancodebogota/auth/controller/AuthenticationController.java

```diff
@@ -0,0 +1,467 @@
+            answer = true;
+        }
+        return answer;
+    }
+
+    public boolean checkAuthenticationOTP(OTPResponseDto answerOtp) {
+        boolean answer = false;
+        if (answerOtp.getStatusCode() != null && answerOtp.getStatusCode().equals("0") && answerOtp.getServerStatusCode().equals("0") && answerOtp.getSeverity().equals("INFO") && answerOtp.getStatusDesc().equals("Operacion Exitosa")) {
+            answer = true;
+        }
+        return answer;
+    }
+
+    public boolean checkCorporateRetrieveCompanyResponseDto(CorporateRetrieveCompanyResponseDto answerRetrieveCompany) {
+        boolean answer = false;
+        if (answerRetrieveCompany.getStatusDto() != null && answerRetrieveCompany.getStatusDto().getStatusCode() != null && answerRetrieveCompany.getStatusDto().getStatusCode().equals(new Long(0L)) && answerRetrieveCompany.getStatusDto().getSeverity().equals("Info") && answerRetrieveCompany.getStatusDto().getStatusDesc().equals("Success")) {
+            answer = true;
+        }
+        return answer;
+    }
+
+    public String validateUserCorporateRequestDto(ValidateUserCorporateRequestDto authRequestDto) {
+        String invalidMessage = "";
+        if (authRequestDto.getLoginId().isEmpty() || authRequestDto.getLoginId().trim().isEmpty()) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "Login invalido. ";
+        }
+
+        if (authRequestDto.getPassword().isEmpty() || authRequestDto.getPassword().trim().isEmpty()) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "Password invalido. ";
+        }
+
+        if (authRequestDto.getIpAddress().isEmpty() || authRequestDto.getIpAddress().trim().isEmpty()) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "IPAddress invalido. ";
+        }
+
+        if (authRequestDto.getCustLoginId().isEmpty() || authRequestDto.getCustLoginId().trim().isEmpty() || authRequestDto.getCustLoginId().length() != 8) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "CustLoginId invalido. ";
+        }
+
+        if (authRequestDto.getOtp().isEmpty() || authRequestDto.getOtp().trim().isEmpty()) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "Otp invalido. ";
+        }
+
+        if (authRequestDto.getCustPermId().isEmpty() || authRequestDto.getCustPermId().trim().isEmpty() || authRequestDto.getCustPermId().length() < 4 || authRequestDto.getCustPermId().length() > 11) {
+            BdbLoggerUtils.info("LOGIN REQUEST INVALID");
+            invalidMessage = invalidMessage + "NIT invalido. ";
+        }
+
+        return invalidMessage;
+    }
+
+    public String actualDate() {
+        DateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

The SimpleDateFormat object was created without setting the time zone. Make sure that you want to use the default time zone or use letter z, Z or X in the pattern. Otherwise, if you don’t call setTimeZone() on the created SimpleDateFormat object, you might get an unexpected date and time when using the object.